### DISCUSSION TITLE
Explain dash in google colab

### DISCUSSION THREAD

**Discussion by JuanBGV** [pycaret/pycaret#2162] [General]
[explain dash.pptx](https://github.com/pycaret/pycaret/files/8064450/explain.dash.pptx)

![explain das](https://user-images.githubusercontent.com/98104321/153959778-89191866-e74c-4e27-a3bc-3cb1d0256f89.png)

I get this error in google colab .. when i use the explain dashboard.
It worked fine for me before. 

Help me, 

Regards

**Comment by ngupta23** [comment]
It would be better if you can just share the Google Colab notebook instead of a PPT and an image. Sharing an image with no code and a small snapshot of the error does not help with debug.

See this link for how to create a reproducible example for GitHub Issues: https://stackoverflow.com/help/minimal-reproducible-example
